docs/user/dashboard/lens.asciidoc

```diff
@@ -19,301 +25,316 @@ data-type="inline"
+If you already know the visualization type you want to use, and how you want to display the data, use the following process:
 
-. Scan through the list of fields to see what’s in your data.
+* *Choose the visualization type.* Open the *Chart type* dropdown, then select the visualization type before you drag any fields.  
 +
-TIP: For sparse datasets, *Empty fields* can contain data, which you can use to create visualizations.
-
-. To filter the data fields, use the following options:
+To view more visualizations that *Lens* automatically created for the fields, click the *Suggestions*. If one of the *Suggestions* meets your visualization needs, click *Save and return* to add it to the dashboard.
 
-* Enter the field name in the *Search field names* field. 
+* *Choose the data you want to visualize.* Drag the fields directly to the layer pane. *Lens* automatically selects the aggregation function.
++
+If you want to learn more about the data a field contains, click *i* next to the field.
 
-* Click *Field by type*, then select the filter. To show all fields in the index pattern, deselect *Only show fields with data*.
+* *Edit and delete.* To change the *Quick function* and display options, click the field in the layer pane. To delete a field, close the configuration options, then click *X* next to the field.
 
 [float]
-[[view-the-data-summaries]]
-===== View the data summaries
+[[change-the-fields]]
+==== Change the fields list
 
-For each field, *Lens* displays a summary depending on the type of data. To analyze the fields, *Lens* uses a sample of 5,000 documents. 
-Each summary displays the percentage of sampled documents over all available documents.
+Change the fields list to display a different index pattern, different time range, or add your own fields.
 
-To view a data summary, click *i* next to the field.
+* To create a visualization with fields in a different index pattern, open the *Change index pattern* dropdown, then select the index pattern. 
 
+* If the fields list is empty, change the <<set-time-filter,time filter>>.
+
+* To add fields, open the action menu (*...*) next to the *Change index pattern* dropdown, then select *Add field to index pattern*.
++
 [role="screenshot"]
-image::images/lens_data_info_documents.png[Data summary analyzed documents]
+image:images/runtime-field-menu.png[Dropdown menu located next to index pattern field with items for adding and managing fields, width=50%]
++
+For more information about adding fields to index patterns and examples, refer to <<runtime-fields>>.
 
-Each data type display the following summaries:
+[float]
+[[lens-formulas]]
+==== Use formulas to perform math
 
-* *Date* &mdash; Displays the time distribution.
+Formulas allow you to perform math on aggregated data. The most common formulas divide two values to produce a percent.
 
-* *String* &mdash; Displays the top 10 values.
+. In the layer pane, click a field.
```

Comment:
This sentence confuses me because "field" typically refers to the "field list" selection pattern, but the way to open formulas is to click the dimension label.